lib/lightning_web/live/audit_live/index.ex

```diff
@@ -47,24 +47,37 @@ defmodule LightningWeb.AuditLive.Index do
   end
 
   def diff(assigns) do
-    rhs = assigns.metadata.after
+    with lhs <- assigns.metadata.before || %{},
+         rhs <- assigns.metadata.after || %{},
+         true <- lhs != %{} || rhs != %{} do
+      changes =
+        lhs
+        |> Map.keys()
+        |> Enum.concat(Map.keys(rhs))
+        |> Enum.sort()
+        |> Enum.uniq()
+        |> Enum.map(fn key ->
+          {key, Map.get(lhs, key), Map.get(rhs, key)}
+        end)
 
-    lhs =
-      assigns.metadata.before ||
-        Enum.into(rhs, %{}, fn {key, _val} -> {key, nil} end)
+      assigns = assign(assigns, changes: changes)
 
-    assigns =
-      assign(assigns,
-        changes:
-          Enum.zip([lhs |> Map.keys(), lhs |> Map.values(), rhs |> Map.values()])
-      )
-
-    ~H"""
-    <%= for {field, old, new} <- @changes do %>
-      <li><%= field %>&nbsp; <%= old %>
-        <Heroicons.arrow_right class="h-5 w-5 inline-block mr-2" />
-        <%= new %></li>
-    <% end %>
-    """
+      ~H"""
+      <.td colspan="4" class="font-mono text-xs break-all">
+        <%= for {field, old, new} <- @changes do %>
+          <li><%= field %>&nbsp; <%= old %>
+            <Heroicons.arrow_right class="h-5 w-5 inline-block mr-2" />
```

Comment:
We are trying to concentrate on a single way of using the icons that is common to the latest Phoenix versions way using the `<.icon name="hero-arrow-right" ...`

Reply:
@jyeshe Thanks - fixed in a follow-up commit.